POKEMON_OF_THE_DAY/README.md

```diff
@@ -0,0 +1,21 @@
+# Pokémon of the Day
+
+
+## Intro
+
+Pokémon Of The Day is a simple integration built to be run on Pandium.  This is the result of Pandium's Pokémon of the Day tutorial.
+
+
+## Integration Functionality
+
+- Allow the user to choose which type of Pokémon they prefer to get a message about.
+- Read the different types of Pokémon available from the pokéAPI to dynamically populate the options for that config.  This will ensure the list is kept up to date even if a new kind of Pokémon is added.
+- When the integration is run it will fetch a Pokémon of the type selected by the user in their connection settings.  Then it will send a Slack message about that Pokémon.  The integration will not send a message about the same Pokémon twice.
```

Comment:
maybe `configuration settings`. connection settings makes me think of network connectivity